README.md

```diff
@@ -108,6 +108,16 @@ then additional inputs/outputs should be submitted to the x-common repository.
 Changes to the test suite (style, boilerplate, etc) will probably have to be made to
```

Comment:
'track specific' might be redundant since this is only talking about the xruby track anyway.